st2reactor/st2reactor/container/sensor_wrapper.py

```diff
@@ -328,7 +328,9 @@ def __init__(self, pack, file_path, class_name, trigger_types,
                                                update_handler=self._handle_update_trigger,
                                                delete_handler=self._handle_delete_trigger,
                                                trigger_types=self._trigger_types,
-                                               queue_suffix='sensorwrapper')
+                                               queue_suffix='sensorwrapper_%s_%s' %
+                                               (self._pack, self._class_name),
+                                               exclusive=True)
```

Comment:
On a related note - eventually we also need to improve TriggerWatcher to utilize routing key since sensors are really only interested in the triggers which belong to that sensor. Right now we do late filtering in the sensor class which is not really efficient if there are many different sensors and triggers running.